modules/apps/product-navigation/product-navigation-user-personal-bar-web/src/main/resources/META-INF/resources/js/index.js

```diff
@@ -92,6 +92,13 @@ export function signInButtonPropsTransformer({
 							document.querySelector('.loading-animation')
 						) {
 							updateModalContent(modalContentHTML);
+							const signInButton =
+								document.getElementsByClassName(
+									'btn disabled btn-primary'
+								)[0];
+
+							signInButton.disabled = false;
+							signInButton.classList.remove('disabled');
```

Comment:
Hey @jgarcialr why do we have 2 checkpoints to run the same code? Can we resort to just using one single JS handler "by the book" from a single resource file? 

Thanks

Reply:
Same code was added to two different checkpoints as adding it only within the script tag wasn't fired by the modal letting the button disabled.
If it was accesed directly pex using /c/portal/login, it worked, that's why I added in here.